### DISCUSSION TITLE
Introduction

### DISCUSSION THREAD

**Discussion by adetyaz** [Open-Science-Community-Saudi-Arabia/Open-Science-Community-in-Saudi#39] [Outreachy 2022] [answered]
Hello my name is adetayo lasisi, an outreachy applicant from Nigeria, I'd love to contribute to this project, specifically on the frontend portion. Please how do I get started.

**Comment by BatoolMM** [comment, marked as answer]
Welcome, @adetyaz to the community!

We have two projects for Outreachy, you can find them here:
1- [Improve the bilinguality and accessibility of the website](https://github.com/Open-Science-Community-Saudi-Arabia/OSCSA_Website/issues/12)
2- Improve Localisation for [Open Educational Resources](https://github.com/Open-Science-Community-Saudi-Arabia/localisation/issues/13).

Some of the tasks in the issue have been addressed - you are also welcome to review open PR!

Let me know if you are still confused!

Feel free to comment under the issue, if you have any questions!

**Comment by adetyaz** [comment]
Hello @BatoolMM thanks for the guide, I've been able to start working on an issue. One thing I noticed is some part of the websites are commented out in the codebase and I'll like to know if we are going to bring them back. Also where can I get the arabic translation for some of the contents  we have?

**Comment by adetyaz** [comment]
@BatoolMM I can't join the OSCSA community, it says quota limit exceeded on jotform
